genesis/genesis.go

```diff
@@ -129,6 +133,7 @@ func FromConfig(config *Config) ([]byte, ids.ID, error) {
 				RewardAddress: destAddr,
 			},
 		)
+		initialSupply += 20 * units.KiloAvax
```

Comment:
Same here: Should we make these constants?

Reply:
Same comment as above